os/tools/configure.sh

```diff
@@ -150,6 +150,14 @@ if [ ! -r "${src_config}" ]; then
   exit 6
 fi
 
+if [ -r ${dest_config} ]; then
+  if [ -x "${TOPDIR}/.version" ]; then
+    echo "Already configured!"
```

Comment:
How about "Already configured and compiled!" to give exact information?

Reply:
yes, I Agree